tcms/issuetracker/base.py

```diff
@@ -15,9 +15,13 @@ class IntegrationThread(threading.Thread):
     def __init__(self, rpc, bug_system, execution, bug_id):
         """
             :param rpc: Bug tracker RPC object
+            :type rpc: dict
             :param bug_system: BugSystem object
+            :type bug_system: dict
```

Comment:
tcms.testcases.models.BugSystem not dict, see Sphinx syntax for making class names links in the docs